public/css/common.less

```diff
@@ -403,3 +403,10 @@ div.show-more {
 form[name="form_confirm_removal"] {
   text-align: center;
 }
+
+.affected-objects {
```

Comment:
this shouldn't affect a list item's height, currently they grow by about 5px in case there is this indicator. Once not all of the items have it (no fake data) this leads to differently sized items, which is bad.